pkg/jsonnet/vm.go

```diff
@@ -26,3 +32,25 @@ func NewVM(param string, importer jsonnet.Importer) *jsonnet.VM {
 	vm.Importer(importer)
 	return vm
 }
+
+func ValidateJSONSchema(name string) *jsonnet.NativeFunction {
+	return &jsonnet.NativeFunction{
+		Name:   name,
+		Params: ast.Identifiers{"schema", "v"},
+		Func: func(s []any) (any, error) {
+			schemaS, err := json.Marshal(s[0])
+			if err != nil {
+				return nil, fmt.Errorf("marshal a JSON Schema as JSON: %w", err)
+			}
+			sch, err := jsonschema.CompileString("schema.json", string(schemaS))
+			if err != nil {
+				return nil, fmt.Errorf("compile a JSON Schema: %w", err)
+			}
+
+			if err = sch.Validate(s[1]); err != nil {
+				return nil, fmt.Errorf("validate data by the JSON Schema: %w", err)
```

Comment:
We should improve the error message.

```console
$ lintnet lint                                           
{
  "lintnet_version": "",
  "env": "darwin/arm64",
  "errors": [
    {
      "message": "evaluate a lint file as Jsonnet: RUNTIME ERROR: validate data by the JSON Schema: jsonschema: '' does not validate with file:///Users/shunsukesuzuki/repos/src/github.com/lintnet/examples/tutorial/validate/schema.json#/required: missing properties: 'name'",
      "lint_file": "main.jsonnet",
      "data_file": "foo.json"
    }
  ]
}
FATA[0000] lintnet failed                                env=darwin/arm64 error=lint failed program=lintnet version=
```

https://github.com/santhosh-tekuri/jsonschema/blob/d3bf05321d6018dbf45a127eb78c7020beb07fe2/errors.go#L92-L99